libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -225,6 +228,32 @@ const AP_Param::GroupInfo AP_Baro::var_info[] = {
 #endif
 #endif  // HAL_BARO_WIND_COMP_ENABLED
 
+#if AP_BARO_THST_COMP_ENABLED
+    // @Param: 1_THST_SCALE
+    // @DisplayName: Thrust compensation
+    // @Description: User provided thrust scaling in Pascals. This is used to adjust linearly based on the thrust output for local pressure difference induced by the props.
+    // @Range: -300 300
+    // @User: Advanced
+    AP_GROUPINFO("1_THST_SCALE", 25, AP_Baro, sensors[0].mot_scale, 0),
+
+#if BARO_MAX_INSTANCES > 1
+    // @Param: 2_THST_SCALE
```

Comment:
i think we should only put it on the first baro, I think this issue can only happen on really tiny vehicles, and fixing just one baro should be enough (just to reduce the params). We could keep the variable in the common structure, but it would just be zero

Reply:
@IamPete1 also suggested upside down props for the test, see if the effect is changed

Reply:
I suspect part of the reason is that the props and motors are already on the underside of the vehicle, pushing away from the FC

Reply:
Tested and fixed